corpus/backtick.txt

```diff
@@ -1,45 +1,83 @@
 ================================================================================
-Simple backtick
+simple backtick
 ================================================================================
 
-`foobar`
+a `foobar` b `:echo`
 
 --------------------------------------------------------------------------------
 
 (help_file
-  (line
-    (backtick
-      (word))))
+  (block
+    (line
+      (word)
+      (backtick
+        (word))
+      (word)
+      (backtick
+        (word)))))
 
 ================================================================================
-Backtick in text
+backtick in text
 ================================================================================
 
-Hello `world`, I am a markup language
+Hello `world`, I am `markup language`. But `this is
+an error`.
 
 --------------------------------------------------------------------------------
 
 (help_file
-  (line
-    (word)
-    (backtick
-      (word))
-    (word)
-    (word)
-    (word)
-    (word)
-    (word)
-    (word)))
+  (block
+    (line
+      (word)
+      (backtick
+        (word))
+      (word)
+      (word)
+      (word)
+      (backtick
+        (word))
+      (word)
+      (word)
+      (ERROR
```

Comment:
Perhaps this should be:

      (backtick
        (word)
        (MISSING "`")))

but arguably, ERROR is correct when someone opens an unclosed (on the same line) backtick?

Reply:
never mind this is fixed by inlining the `$.word2` cases into `$.word`. 🤷